vpc/tool/container2/setup_container_linux.go

```diff
@@ -225,13 +208,13 @@ func addIPv4AddressAndRoute(ctx context.Context, nsHandle *netlink.Handle, link
 	err = nsHandle.RouteDel(&oldLinkRoute)
 	if err != nil {
 		logger.G(ctx).WithError(err).Error("Unable to delete route on link")
-		return errors.Wrap(err, "Unable to add delete existing 'link' route to link")
+		return 0, errors.Wrap(err, "Unable to add delete existing 'link' route to link")
 	}
 
-	return nil
+	return mtu, nil
```

Comment:
This return of mtu from this function confuses me a little bit.
I get it though, this is exactly where the actual default route mtu is calculated, and needs to be consumed later on.

But would a function called `getDefaultRouteMtu()` (to be injected into setupHTBClasses) that does this make the code any simpler?

Reply:
I feel like then we're writing the code to see if a default route exists & returning the MTU in two different places, and that seems a little weird to me?